src/components/stats/VolumeChart.tsx

```diff
@@ -194,12 +193,20 @@ const useVolumeMetrics = (): ({ timestamp: number } & Record<
             cumulativeRedeemed,
             cumulativeRedeemedMasset,
             cumulativeFeesPaid,
-            savingsContracts: [{ cumulativeDeposited, cumulativeWithdrawn }],
+            savingsContracts,
           } = values;
 
+          const collectiveDeposited = savingsContracts.reduce(
```

Comment:
This will have to be a `let` and re-run to subtract the `prev` value (see below)